cms/static/sass/elements-v2/_header.scss

```diff
@@ -0,0 +1,181 @@
+// studio - elements - global header
+// ====================
+
+.wrapper-header {
+  position: relative;
+  width: 100%;
+  box-shadow: 0 1px 2px 0 $shadow-l1;
+  margin: 0;
+  padding: 0 $baseline;
+  background: $white;
+
+  header.primary {
+    @include clearfix();
+    box-sizing: border-box;
+    max-width: $fg-max-width;
+    min-width: $fg-min-width;
+    width: flex-grid(12);
+    margin: 0 auto;
+  }
+
+  // ====================
+
+  // basic layout
+  .wrapper-l, .wrapper-r {
+    background: $white;
+  }
+
+  .wrapper-l {
+    @include float(left);
+    width: flex-grid(7,12);
+  }
+
+  .wrapper-r {
+    @include float(right);
+    @include text-align(right);
+    width: flex-grid(4,12);
+  }
+
+  .branding, .info-course, .nav-course, .nav-account, .nav-pitch {
+    box-sizing: border-box;
+    display: inline-block;
+    vertical-align: middle;
+  }
+
+  .user-language-selector {
+    width: 120px;
+    display: inline-block;
+    margin: 0 10px 0 5px;
+    vertical-align: sub;
+
+    .language-selector {
+      width: 120px;
+    }
+  }
+
+  .nav-account {
+    width: auto;
+  }
+
+  // basic layout - nav items
+  nav {
+
+    > ol > .nav-item {
+      display: inline-block;
+      vertical-align: middle;
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      h3{
+        font-size: font-size(base);
+        font-weight: font-weight(semi-bold);
+      }
+    }
+
+    .nav-item a {
```

Comment:
This styles covers a lot of links in the header. The styles span across multiple template files as well. I say we leave this be for now to lower the risk of style breakage and work spent here.